### ISSUE TITLE
Dedicated playlist page for public playlists

### ISSUE THREAD

**Issue by ddevault** [Chocobozzz/PeerTube#2672]
I like the playlist overview page which I can access for playlists on my account (click playlists in the sidebar, then select a playlist), but this page is private to my login session. It does not seem like a similar page is available for playlists I've made public on my channel, instead the only option seems to a video playback page with the playlist in the sidebar.

**Comment by Chocobozzz** [comment]
Hello,

There is a dedicated page for your channel public playlists: https://peertube2.cpy.re/video-channels/a75cbdf4-acf2-45c0-a491-e9b28939f8db/video-playlists

**Comment by ddevault** [comment]
That page is for the list of playlists - not for the playlists themselves. To clarify, I don't want a page like this:

![List of playlists](https://l.sr.ht/m5EM.png)

But like this:

![The playlist itself](https://l.sr.ht/qlJJ.png)

**Comment by kamazeuci** [comment]
+1

**Comment by PleaseStateTheNatureOfYourInquiry** [comment]
Indeed. The list of available playlists is good and handy! It would be fabulous if, when a visitor clicks on a playlist, she sees the overview of that playlist: the thumbnail and the description on the left, the available videos on the right, just like the owner of a playlist can see. 
Here is an example of how the owner can see a playlist:

<img width="1154" height="647" alt="Image" src="https://github.com/user-attachments/assets/705c4aae-36b5-429e-b0d9-1a06bb80a32c" />

and here how a non-registered visitor will see it:

<img width="1698" height="653" alt="Image" src="https://github.com/user-attachments/assets/ce9e74a3-b972-437c-b9fe-f4736126f171" />

(https://filmspace.lightcurvefilms.com/w/p/7KAPaPRWjJAGD9Jfpy1tvZ?playlistPosition=1)

Thanks for any advice!
